docs/guidelines/content-guidelines/01-user-docs.md

```diff
@@ -1,49 +1,77 @@
 
-1. In the `/kyma` repository, open the<!-- markdown-link-check-disable-line --> [`/kyma/docs/_sidebar.md`](https://github.com/kyma-project/kyma/blob/main/docs/_sidebar.md) file and under **Modules**, add a link to your module documentation main page.
-2. Update `index.html` by adding a new **alias** entry in the **<body>** part. Use the following pattern:
+1. In the `/kyma` repository, open the<!-- markdown-link-check-disable-line --> [`/kyma/vitepress/config.mjs`](https://github.com/kyma-project/kyma/blob/main/.vitepress/config.mjs).
+2. Add `import {YOUR_MODULE_NAME}Sidebar from '../docs/external-content/{YOUR_MODULE_NAME}/docs/user/_sidebar';` as the next import line.
+3. Provide your module details in the `sidebar` section, under `themeConfig`. Use the following pattern:
 <!-- markdown-link-check-disable -->
-    ```html
-    alias: {
-    '/{MODULE_REPOSITORY_NAME}/(.*)': 'https://raw.githubusercontent.com/{GITHUB_PROJECT_NAME}/{MODULE_REPOSITORY_NAME}/main/docs/$1',
-    }, 
-    ```
-
-    For example:
-
-    ```html
-    alias: {
-    '/serverless-manager/(.*)': 'https://raw.githubusercontent.com/kyma-project/serverless-manager/main/docs/$1',
-    }, 
-    ```
+   ```mjs
+   {
+      text: 'My Module',
+      link: '/external-content/my-module/docs/user/README.md',
+      collapsed: true,
+      items: makeSidebarAbsolutePath(
+        myModuleSidebar,
+        'my-module',
+      ),
+   },
+   ```
+
+   If you want to add documentation for a new module, you must also add the module repository name do the `deploy.yml` file, under `jobs.copy-docs.strategy.matrix.repository`. For example:
+
+   ```yaml
+   jobs:
+     copy-docs:
+       strategy:
+         matrix:
+           repository:
+             - btp-manager
+             - istio
+             - serverless
+             - {YOUR_MODULE_REPO}
+   ```
 <!-- markdown-link-check-enable -->
-> **CAUTION:** When you update navigation paths in documentation, make sure you check all `_sidebar.md` files that may be affected.
+> **CAUTION:** When you update navigation paths in documentation, make sure you check all `_sidebar.ts` files that may be affected.
+
+4. Run the [Deploy VitePress site to GitHub Pages](https://github.com/kyma-project/kyma/actions/workflows/deploy.yml) or wait for the CronJob to start it (every day at midnight).
+5. Make sure that the `public` folder is deleted after the build. If not, delete it manually to clean up the environment.
+
+## Execute Prettier
+
+Prettier helps maintain proper formatting. The project is already configured to use this formatter automatically, provided you are using Visual Studio Code (it leverages the VSCode Prettier plugin). Unfortunately, it runs prettier at commit time on Git; if you are not using Git integrated in VSCode for commits, you must execute it manually by running the following command:
```

Comment:
I'm not sure if I understand. So if I'm using Visual Studio Code, do I need to install the VSC Prettier plugin?

Reply:
I'll rephrase that.